qemu/tests/cfg/viorng_in_use.cfg

```diff
@@ -3,32 +3,37 @@
     start_vm = yes
     kill_vm_on_error = yes
     check_guest_bsod = yes
-    login_timeout = 240
+    login_timeout = 360
     suppress_exception = no
     run_bgstress = rng_bat
-    session_cmd_timeout = 240
-    read_rng_cmd = "WIN_UTILS:\random_%PROCESSOR_ARCHITECTURE%.exe"
-    rng_data_rex = "0x\w"
-    driver_name = "viorng"
+    session_cmd_timeout = 360
     cdrom_cd1 = isos/windows/winutils.iso
     no no_virtio_rng
-    only Windows
+    Windows:
+        driver_id_pattern = "(.*?):.*?VirtIO RNG Device"
+        target_process = random\w*.exe
+        rng_data_rex = "0x\w"
+        driver_name = "viorng"
+        read_rng_cmd = for /l %i in (1, 1, 1000) do "WIN_UTILS:\random_%PROCESSOR_ARCHITECTURE%.exe"
         i386:
             driver_id_cmd = 'WIN_UTILS:\devcon\wxp_x86\devcon.exe find * | find "VirtIO"'
             driver_check_cmd = "WIN_UTILS:\devcon\wxp_x86\devcon.exe status @DRIVER_ID"
-            driver_id_pattern = "(.*?):.*?VirtIO RNG Device"
         x86_64:
             driver_id_cmd = 'WIN_UTILS:\devcon\wnet_amd64\devcon.exe find * | find "VirtIO"'
             driver_check_cmd = "WIN_UTILS:\devcon\wnet_amd64\devcon.exe status @DRIVER_ID"
-            driver_id_pattern = "(.*?):.*?VirtIO RNG Device"
+    Linux:
+            read_rng_cmd  = "dd if=/dev/random bs=10 count=1000000 2>/dev/null|hexdump"
```

Comment:
I have a double confirm：
1.If host cpu does not have rdrand flag and guest rng backend is /dev/random,the read_rng_cmd will take very very long time;
2.But if guest rng backend is /dev/urandom,this cmd will not take too long time,only about 2+mins.
3.Now rng backend is already changed to /dev/urandom,so I think there should be no big issue.
4.I try again on an amd host,repeat during case five times,all passed:
(1/5) smp_1.512m.repeat1.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.viorng_in_use.with_stop_continue.during_bg_test: PASS (504.63 s)
 (2/5) smp_1.512m.repeat2.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.viorng_in_use.with_stop_continue.during_bg_test: PASS (299.48 s)
 (3/5) smp_1.512m.repeat3.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.viorng_in_use.with_stop_continue.during_bg_test: PASS (303.91 s)
 (4/5) smp_1.512m.repeat4.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.viorng_in_use.with_stop_continue.during_bg_test: PASS (308.63 s)
 (5/5) smp_1.512m.repeat5.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.viorng_in_use.with_stop_continue.during_bg_test: PASS (300.64 s)

@suqinhuang